### ISSUE TITLE
DataFrame with float columns can't be loaded correctly in CASTable

### ISSUE THREAD

**Issue by gygabyte017** [sassoftware/python-swat#59]
If a column of a dataframe is float, when the frame is uploaded to sas the numbers become totally wrong (the period is just removed)

Example:
```
df = pd.DataFrame(data=[1.2], columns=['Col1'])
conn.read_frame(df,{'name': 'XXX', 'caslib': 'XXX', 'promote': True})
```
On the cas table I read that Col1=12 and not Col1=1.2

How to fix this?

**Comment by kesmit13** [comment]
I'm not able to reproduce this.  Can you verify that `df.to_csv(index=False, header=False)` is generating the correct data?  Also, can you send the version of Python and SWAT (swat.__version__) that you are using?

**Comment by kesmit13** [comment]
That should be `swat.__version__` in the previous comment.

**Comment by gygabyte017** [comment]
Python version is 3.6.5 and Swat version is '1.5.2'.

I found out the issue: my system has regional settings where the comma ',' is the decimal separator. In python in general the regional settings are not used, in fact if I print the dataframe I see "Col1=1.2" and not "Col1=1,2".

However, it seems that the "to_csv" is using regional settings, so with "read_frame" the value "1.2" is first converted to "1,2" by "to_csv" and then is converted to "12" by the cas upload.

I solved it with
`conn.upload_frame(df, casout={...}, importoptions={'locale':'EN-us'})`
